scripts/lib/CIME/case.py

```diff
@@ -1108,6 +1108,19 @@ def apply_user_mods(self, user_mods_dir=None):
                 user_mods_path = os.path.join(user_mods_path, user_mods)
             apply_user_mods(self._caseroot, user_mods_path)
 
+    def _get_comp_user_mods(self, component):
+        """
+        For a component 'foo', gets the value of FOO_USER_MODS.
+
+        Returns None if no value was found, or if the value is an empty string.
+        """
+        comp_user_mods = self.get_value("{}_USER_MODS".format(component.upper()))
+        #pylint: disable=no-member
```

Comment:
This is definitely cleaner. But I was hesitant to bypass this pylint check, figuring that pylint might be seeing some real possibility for problems. In particular: If there is any possibility that comp_user_mods is **not** a string here, then this code will raise an exception. I'll defer to your understanding here: If you're confident that get_value will **always** return a string here, then I'm happy with your change.